h5p-bildetema/semantics.json

```
@@ -52,6 +52,12 @@
         "name": "footerLink3Href",
         "default": "https://lexin.oslomet.no/",
         "type": "text"
+      },
+      {
+        "label": "Breadcrumbs 'Topic' label",
+        "name": "breadcrumbsTopic",
+        "default": "Tema",
```

The default should be in English

```suggestion
        "default": "Topic",
```

---

👍